src/seedu/addressbook/AddressBook.java

```diff
@@ -207,14 +207,25 @@
      */
 
     public static void main(String[] args) {
-        showWelcomeMessage();
-        processProgramArgs(args);
-        loadDataFromStorage();
+    	showToUser(DIVIDER, DIVIDER, VERSION, MESSAGE_WELCOME, DIVIDER);
```

Comment:
Good work by inlining a good number of methods into main(). SLAP is now violated :-)